### PR TITLE
Update ReleaseNotes for 2.6.8

### PR DESCRIPTION
**Pull request by ChrisBr**

### CODE REVIEW

**Comment by ChrisBr** [comment]
@hennevogel cherry picked your commit to fix CI

File: ReleaseNotes-2.6.8 [hunk 1]
```diff
@@ -21,12 +21,13 @@ Bugfixes:
...
-* [webui] Fixes CVE-2016-0752: Possible Information Leak Vulnerability in Action View
-* [webui] Fixes CVE-2016-0753: Possible Input Validation Circumvention in Active Model
-* [webui] Fixes CVE-2015-7581: Object leak vulnerability for wildcard controller routes in Action Pack
+* [webui] Update rails to version 4.1.14.1 to fix CVE-2015-7576: Timing attack vulnerability in basic authentication in Action Controller.
+* [webui] Update rails to version 4.1.14.1 to fix CVE-2016-0751: Possible Object Leak and Denial of Service attack in Action Pack
+* [webui] Update rails to version 4.1.14.1 to fix CVE-2015-7577: Nested attributes rejection proc bypass in Active Record.
+* [webui] Update rails to version 4.1.14.1 to fix CVE-2016-0752: Possible Information Leak Vulnerability in Action View
+* [webui] Update rails to version 4.1.14.1 to fix CVE-2016-0753: Possible Input Validation Circumvention in Active Model
+* [webui] Update rails to version 4.1.14.1 to fix CVE-2015-7581: Object leak vulnerability for wildcard controller routes in Action Pack
+* [webui] Fix redirect after login for iChain and proxy mode
```
Comment (hennevogel) [inline ReleaseNotes-2.6.8]: ReleaseNotes are for OBS Admins so I this should be enough:

* [webui] Update rails to version 4.1.14.1 to fix several security issues (CVE-2015-7576, CVE-2016-0751, CVE-2015-7577, CVE-2016-0752, CVE-2016-0753, CVE-2015-7581)
*  [webui] Fix redirect after login for iChain and proxy mode

**Comment by mdeniz** [comment]
LGTM beside the nitpicks :+1:

**Comment by ChrisBr** [comment]
Okay!

**Pull request merged**